rouille_proc_macro/src/lib.rs

```diff
@@ -83,6 +83,58 @@ rouille_compilogenese::rouille! {
             "Ponctuation" => "Punct",
             "Litéral" => "Literal",
             "macro_procédurale" => "proc_macro",
+            "LectureBufferisée" => "BufRead",
+            "Lecture" => "Read",
+            "Écriture" => "Write",
+            "entrée_sortie" => "io",
+            "entrée" => "stdin",
+            "sortie" => "stdout",
+            "lire_ligne" => "read_line",
+            "écrire_ligne" => "write_line",
+            "écrire" => "write",
+            "est_vide" => "is_empty",
+            "est" => "is",
+            "verrouiller" => "lock",
+            "octets" => "bytes",
+            "chaîne" => "str",
+            "lire_exactement" => "read_exact",
+            "lire_jusqu'à" => "read_until",
+            "sépare" => "split",
+            "en_tant_que" => "as",
+            "Entier" => "Integer",
+            "ensuite" => "then",
+            "ensuite_valeur" => "then_some",
```

Comment:
Le maintien de la cohérence est une des prérogatives majeures de l'académie rouillesque.

```suggestion
            "ensuite_quelque" => "then_some",
```